Makefile

```diff
@@ -228,43 +214,21 @@ else
 endif
 
 compile_lib: config qemu/config-host.h-timestamp
-	cd qemu && $(MAKE) -j 4
-	$(MAKE) unicorn
+	$(MAKE) -C qemu -j 4
 
-unicorn: $(LIBRARY) $(ARCHIVE)
+unicorn: compile_lib $(LIBRARY) $(ARCHIVE)
 
-$(LIBRARY): $(UC_TARGET_OBJ) uc.o list.o
-ifeq ($(UNICORN_SHARED),yes)
-ifeq ($(V),0)
```

Comment:
I'm actually still trying to move over to autotools. since autotools handles verbosity, i think it'd be a better option (https://autotools.io/automake/silent.html)